render/topologies.go

```diff
@@ -82,8 +82,19 @@ var ProcessNameRenderer = Map{
 // graph by merging the process graph and the container topology.
 var ContainerRenderer = MakeReduce(
 	Map{
-		MapFunc:  MapProcess2Container,
-		Renderer: ProcessRenderer,
+		MapFunc: MapProcess2Container,
+		Renderer: MakeReduce( // We only want processes in container _or_ processes with network connections
+			FilterUnconnected{
+				Renderer: ProcessRenderer,
+			},
```

Comment:
This appears to be a change in the behavior, previously we weren't filtering unconnected, now we are?